docs/markdown/getting-started.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started
+
+This tutorial will help you start an Interledger node locally on you computer and create a local ILP test network.
+
+This tutorial describes how to:
+1. Install [`moneyd`](../concepts/moneyd.md) on your system
+2. Start an [Interledger](https://github.com/interledgerjs/ilp-connector) node on a local test network using moneyd
+3. Send and receive value using the [SPSP](../concepts/spsp.md) (Simple Payment Setup Protocol) API
+
+![Getting-Started-Locally](../images/getting-started-locally.png)
+
+## Before you begin
+* Install a stable version of [Node.js](https://nodejs.org/en/) (10.16.0 LTS is recommended)
+* Set up Node to install modules globally <br/>
+
+NOTE: For this tutorial you *do not* need to use any cryptocurrency.
+
+## Installing moneyd
+To install `moneyd`, open a command line and run the following command:
+```shell
+$ npm install -g moneyd
+```
+
+## Starting a node
+After you’ve installed `moneyd`, run the following command to start your local node:
+```shell
+$ moneyd local
+```
+Running the above command creates an Interledger node that listens on port 7768.
+
+## Sending and receiving value
+Once you have `moneyd` running, you can send and receive value over ILP using the SPSP API. For this tutorial, 
+we’ll use the SPSP command line tool.
```

Comment:
Might be worth noting that since we're running in local mode right now we'll only be able to send to ourselves for the moment

Reply:
I have a diagram at the beginning of the tutorial that shows this. Sadly, HTML is not able to show the image.

https://github.com/pgarg-ripple/interledger.github.io/blob/pgarg-ilp/docs/markdown/getting-started.md

Reply:
Why not? You should be able to embed the image, no?

Reply:
I tried :-(  I am giving the right path.. couldn't figure out. So commented it out for now.